splitio/models/telemetry.py

```diff
@@ -27,7 +27,7 @@ class CounterConstants(Enum):
     EVENTS_QUEUED = 'eventsQueued'
     EVENTS_DROPPED = 'eventsDropped'
 
-class ConfigParams(Enum):
+class _ConfigParams(Enum):
```

Comment:
why classes with `_` at the begining?

Reply:
The constants class is used only internal in telemetry model.